packages/app/src/assets/icons/chevron-down.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20" fill="#ffffff">
+  <path fill-rule="evenodd" d="M5.293 7.293a1 1 0 011.414 0L10 10.586l3.293-3.293a1 1 0 111.414 1.414l-4 4a1 1 0 01-1.414 0l-4-4a1 1 0 010-1.414z" clip-rule="evenodd" />
+</svg>
```

Comment:
According to the other component you could use [Font Awesome](https://fontawesome.com/icons?d=gallery&q=dropdown).

Reply:
I just tried replacing the `img` tags in `Dropdown.vue` with `<i class="fas fa-angle-down"></i>` from fontawesome but that breaks the on click behavior of the entire component (I don't know why or how). So I will stick with the current solution